fearless/ba-RU.lproj/Localizable.strings

```diff
@@ -237,7 +237,6 @@
 "staking.payouts.learn.more" = "Узнать больше про выплаты";
 "staking.reward.payout.account" = "Награда аккаунты";
 "staking.selected.accounts.mask" = "Аккаунттар(%d)";
-"staking.recommended.validators.title" = "Кәңәшләнгән валидаторҙар";
```

Comment:
Whe we need to remove localization for different screen?

Reply:
This is weird, I just updated all the files from Lokalize

Reply:
Will upload it to Lokalize

Reply:
No, I won't upload it. Seems like it's a wrong key and we need to remove it one day